src/modules/shared/pages/innovation/support/support-summary-list.component.ts

```diff
@@ -113,6 +115,13 @@ export class PageInnovationSupportSummaryListComponent extends CoreComponent imp
             : ''
         }));
 
+        if (
+          this.sectionsList[0].unitsList.length ||
+          this.sectionsList[1].unitsList.length ||
+          this.sectionsList[2].unitsList.length
```

Comment:
If you prefer, I think something like this would work:
```suggestion
          this.sectionsList.some(s => s.unitsList.length > 0)
```